services/sharedchannel/channelinvite.go

```diff
@@ -98,7 +98,7 @@ func combineErrors(err error, serror string) string {
 	return sb.String()
 }
 
-func (scs *Service) onReceiveChannelInvite(msg model.RemoteClusterMsg, rc *model.RemoteCluster, response remotecluster.Response) error {
+func (scs *Service) onReceiveChannelInvite(msg model.RemoteClusterMsg, rc *model.RemoteCluster, _ remotecluster.Response) error {
```

Comment:
Can we change the method signature altogether to not pass the param? Or is is satisfying some interface?

Reply:
Yeap it conforms to function type https://github.com/mattermost/mattermost-server/blob/MM-27493_shared_channels/services/remotecluster/service.go#L55